crates/nu-command/src/strings/detect_columns.rs

```diff
@@ -44,6 +44,48 @@ impl Command for DetectColumns {
     ) -> Result<PipelineData, ShellError> {
         detect_columns(engine_state, stack, call, input)
     }
+
+    fn examples(&self) -> Vec<Example> {
+        let span = Span::test_data();
+        vec![
+            Example {
+                description: "Splits string across multiple columns",
+                example: "echo 'a b c' | detect columns -n",
+                result: Some(Value::List {
+                    vals: vec![Value::Record {
+                        cols: vec![
+                            "Column0".to_string(),
+                            "Column1".to_string(),
+                            "Column2".to_string(),
+                        ],
+                        vals: vec![
+                            Value::test_string("a"),
+                            Value::test_string("b"),
+                            Value::test_string("c"),
+                        ],
+                        span,
+                    }],
+                    span,
+                }),
+            },
+            Example {
```

Comment:
This example test failed, anyone give me some suggestions?

Reply:
You may need to write it without using string interpolation. Tests only let you use a small number of commands.

Maybe this will work?

```
"c1 c2 c3" + (char nl) + "a b c" | detect columns
```

Reply:
@jntrnr Very strange, not work either.
```
failures:

---- strings::detect_columns::test::test_examples stdout ----
input: echo 'a b c' | detect columns -n
result: List { vals: [Record { cols: ["Column0", "Column1", "Column2"], vals: [String { val: "a", span: Span { start: 15, end: 29 } }, String { val: "b", span: Span { start: 15, end: 29 } }, String { val: "c", span: Span { start: 15, end: 29 } }], span: Span { start: 15, end: 29 } }], span: Span { start: 0, end: 0 } }
done: 758.324µs
thread 'strings::detect_columns::test::test_examples' panicked at 'test eval error in `"c1 c2 c3" + (char nl) + "a b c" | detect columns`: ExternalNotSupported(Span { start: 46, end: 50 })', crates/nu-command/src/example_test.rs:125:25
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace


failures:
    strings::detect_columns::test::test_examples

test result: FAILED. 320 passed; 1 failed; 0 ignored; 0 measured; 0 filtered out; finished in 1.11s
```

Reply:
Oh maybe it's the `char nl` that's not supported. It's okay to set this answer to None for now.